packages/util-eslint-runner/bin/index.js

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env node
+'use strict';
+
+const spawn = require('child_process').spawn;
+const fs = require('fs');
+const glob = require('glob');
+const program = require('commander');
+
+program
+	.option('-n --name [string]', 'NPM script to execute', null)
+	.option('-p --pattern [string]', 'Glob pattern to search', '**/*.js')
+	.parse(process.argv);
+
+/**
+ * Search for javascript files
+ * @param {Object} arg
+ */
+const globbing = arg => {
+	glob(program.pattern, {ignore: arg}, (err, paths) => {
+		if (err) {
+			throw err;
+		}
+		if (paths.length > 0 && program.name) {
+			// If javascript found run script
+			spawn('npm', ['run', program.name], {stdio: 'inherit'});
+		}
+	});
+};
```

Comment:
You can hook into the `found` event and then exit searching early.

```
const globSearch = glob(program.pattern, {ignore: arg}, (err) => {
   if (err) {
       throw err;
   }
});

globSearch.on('found', (file) => {
   globSearch.abort();
   spawn('npm', ['run', program.name], {stdio: 'inherit'});
});
```

Reply:
oh yeah, nice 👍